### ISSUE TITLE
[QST] Preferences on stand in Dask-SQL logo

### ISSUE THREAD

**Issue by taureandyernv** [dask-contrib/dask-sql#342]
Hey, I was working on a Dask-SQL portion of a webpage that also had a separate Dask portion and logo.  In lieu of an official logo, I created these 3 possible logo options for Dask-SQL based on the Exo font and your alterations to it.  The Q's style had some debate to it. 
We wanted to know which do you prefer to be a temporary logo?
![dasksql](https://user-images.githubusercontent.com/46935140/145273896-386099b0-c000-431d-88d9-70e01b8d7d37.png)
![dasksql2](https://user-images.githubusercontent.com/46935140/145273987-3500d391-89ed-4b62-b88b-2a482a133a64.png)
![dasksql3](https://user-images.githubusercontent.com/46935140/145274005-e0e673a7-081d-4a06-a8ff-d70217f111e9.png)

@randerzander @quasiben @exactlyallan @jacobtomlinson

**Comment by randerzander** [comment]
Thanks for creating some example logos!

I'm not sure what rules or guidelines Dask has (if any) for dask-contrib projects. That said, I personally like either of the last two (can't actually tell how they are different?)

**Comment by taureandyernv** [comment]
just the tail of the Q is different

**Comment by jacobtomlinson** [comment]
I think having logos for subprojects is a great idea. We would be able to use them in documentation to make it clear which site you are on. 

Of those examples I think my preference would be for the last one.

cc @jsignell and @jrbourbeau who I've discussed Dask branding with lately.

**Comment by jsignell** [comment]
Yeah :+1: on sub-project logos. From an org perspective, it would be great to come up with guidelines on how to make these. So once you settle on one (2 and 3 both seem good) can you add a section to https://marketing.dask.org/en/latest/colors.html#logos explaining how you did it?
